src/routes.ts

```diff
@@ -7,6 +7,7 @@ enum RoutePath {
   PRIZE = '/prize-tap',
   ABOUT = '/about',
   NFT = '/pass',
+  LEADERBOARD = '/leader-board',
```

Comment:
We would love it as /leaderboard. You can use the Figma design file and layout names for naming elements.